plugins/action_update_review_status/src/updateReviewStatuses.ts

```diff
@@ -60,8 +60,6 @@ const updateReviewStatuses: ActionPluginType = async ({
     const reviews = (await db.getAssociatedReviews(applicationId, stageNumber))
       // Ignore all "Discontinued" reviews
       .filter(({ reviewStatus }) => reviewStatus !== ReviewStatus.Discontinued)
-      // Ignore current review
-      .filter(({ reviewId: id }) => id !== reviewId)
```

Comment:
Does this not work for application triggers if it's here?

Reply:
It is needed because on line 109 it needs to get the `current review level`: const thisReviewLevel = reviews.find((review) => review.reviewId === reviewId)?.levelNumber